cloud-controller-manager/do/loadbalancers.go

```diff
@@ -40,6 +40,10 @@ const (
 	// used to enable fast retrievals of load-balancers from the API by UUID.
 	annoDOLoadBalancerID = "kubernetes.digitalocean.com/load-balancer-id"
 
+	// annDOLoadBalancerName is the annotation used to specify a name of the
+	// load balancer that is going to be created by the controller.
+	annDOLoadBalancerName = "service.beta.kubernetes.io/do-load-balancer-name"
```

Comment:
Interesting! Are the other `service.beta.kubernetes.io/do-*` official annotations @timoreimann? I couldn't find them in official docs. If there annotations are not official too, I wonder what is the good balance between consistency / confusion :thinking:

Reply:
Ah you're totally right, we should stick to the annotation you have chosen in this case: I saw `kubernetes.digitalocean.com/load-balancer-id` above and only realized now that we picked a different prefix in that one case because that annotation is meant to be read-only. The LB name is to be created/modified by users for sure.

Thanks for double-checking on this one -- let's keep it the way it is. 👍